docs/running-in-production.md

```diff
@@ -193,3 +193,14 @@ proposing the next block).
 
 By default, the maximum number of transactions per a block is 10_000.
 Feel free to change it to suit your needs.
+
+-   `p2p.addr_book_strict`
+
+By default, Tendermint checks whenever a peer's address is routable before
+saving it to the address book. The address is considered as routable if the IP
+is [valid and within allowed
+ranges](https://github.com/tendermint/tendermint/blob/master/p2p/netaddress.go#L207).
```

Comment:
use https://github.com/tendermint/tendermint/blob/27bd1deabe4ba6a2d9b463b8f3e3f1e31b993e61/p2p/netaddress.go#L207

because code changes might make that line number wrong. I used [this feature](https://help.github.com/articles/getting-permanent-links-to-files/#press-y-to-permalink-to-a-file-in-a-specific-commit)